app/src/main/res/layout/activity_entry_image.xml

```diff
@@ -42,27 +45,47 @@
             android:layout_below="@id/entry_image_question_text_view"
             android:layout_marginBottom="@dimen/activity_vertical_margin"
             android:text="@string/entry_image_description"
+            android:textSize="@dimen/entry_submit_image_description_textsize"
+            android:textColor="@color/colorEntrySubmitImageDescription"
             />
 
-        <Button
-            android:id="@+id/entry_image_camera_button"
+        <android.support.v7.widget.CardView
+            android:id="@+id/entry_image_camera_button_card_view"
+            style="@style/AppTheme.IconButtonCardView"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_below="@id/entry_image_description_text_view"
-            android:text="@string/upload_image_camera"
             app:layout_widthPercent="48%"
-            />
+            >
+
+            <Button
```

Comment:
Removing the card padding (and adding it to the button instead) seems to do the trick. This button style is particularly big, and users overall avoid clicking in corners in my experience.